Riot/Views/RoomTitle/PreviewRoomTitleView.xib

```diff
@@ -6,7 +6,6 @@
     <dependencies>
```

Comment:
I would prefer update `PreviewRoomTitleView` class later in another PR.
Please remove your changes for the moment.